examples/chartist.js/dashboard-app/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "chartistjs-example",
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "dev": "parcel src/index.html",
+    "without-cube": "parcel src/without-cube/index.html",
+    "build": "parcel build src/index.html"
+  },
+  "devDependencies": {
+    "parcel": "^2.6.2",
+    "process": "^0.11.10"
```

Comment:
Why is this in dependencies?

Reply:
@cube-dev/example-wrapper requires it

Reply:
Hmm! @ewgenius, does `@cube-dev/example-wrapper` really require any additional deps?